### ISSUE TITLE
Split OpenAPI spec into several files

### ISSUE THREAD

**Issue by behas** [graphsense/graphsense-openapi#4]
Smaller files make the API easier to read / understand.

https://davidgarcia.dev/posts/how-to-split-open-api-spec-into-multiple-files/

**Comment by myrho** [comment]
I agree. In the beginning the spec was split, but then it turned out that having the spec in one file is easier to process by various generators.
